### ISSUE TITLE
Incorrect icons for New File and Change Sort Order

### ISSUE THREAD

**Issue by raiwh** [whyt-byte/Blue-Topaz_Obsidian-css#100]
The icons for New File and Change Sort Order are not appropriate for the function they represent.  The default icon for New File is a question mark and the default for Change Sort Order is the Internet Explorer Logo.  On hover these change to a Heart Icon (new file) and the Facebook Logo (change sort order).  See attached images

<img width="238" alt="Default icon set - no hover" src="https://user-images.githubusercontent.com/69487264/140517735-76cbbe0a-e90a-4b57-bc0b-691fdf549205.png">

<img width="248" alt="new file on hover" src="https://user-images.githubusercontent.com/69487264/140517800-68d09176-2498-4a4a-bdbf-70c23e5739f4.png">

<img width="256" alt="Change Sort Order On hover" src="https://user-images.githubusercontent.com/69487264/140517834-2e2579f2-25ab-4634-9f0b-58b071a5cb7c.png">

**Comment by whyt-byte** [comment]
It's bizarre, indeed! 😂 However, I have no idea about the bug because I cannot reproduce it. I have modified some lines of CSS, but I am not sure whether it works. Could you please update the theme to find out if this bug is fixed? Thank you for your help.

By the way, shall I know which language you choose for your Obsidian? As I am trying to support all languages, I'm not sure whether this is caused by the language selected.

**Comment by raiwh** [comment]
I restarted Obsidian and the icon issue has been resolved.  I did not update the theme but simply restarted - so not sure what happened.  I have restarted many times since the icon issue emerged and done multiple CSS checks to find issue.  But today all is good so I am grateful.  Yes I do believe the issue may be related to the languages/fonts.  I have not adjusted any fonts and use English in Obsidian.  Anyway the bug appears fixed so whatever you did it worked.  Thank you
